db/seeds/dummy_structures.rb

```diff
@@ -51,6 +73,14 @@
     Partnership.find_or_create_by!(school: school, lead_provider: LeadProvider.first, cohort: Cohort.find_or_create_by!(start_year: 2022))
   end
 
+  user = User.find_or_create_by!(email: "second-school-leader@example.com") do |u|
+    u.full_name = "School Leader User - Induction Coordinator"
+    u.confirmed_at = Time.zone.now.utc
+  end
+  if School.any?
+    InductionCoordinatorProfile.joins(:schools).find_or_create_by!(user: user, schools: [School.first])
+  end
+
```

Comment:
Do we need this as well as the numbered ones?

Reply:
Modified it.